### PR TITLE
chore: Add optional parameter for chrome path

### PR DESCRIPTION
**Pull request by AhmedAbdoOrtiga**
#### Description of changes

Add optional parameter for chrome path so the action can pass it, it will be used only for the library.

#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Added relevant unit test for your changes. (`yarn test`)
- [ ] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [ ] Ran precheckin (`yarn precheckin`)

### CODE REVIEW

File: packages/cli/src/scanner/page.ts [hunk 1]
```diff
@@ -20,9 +20,9 @@ export class Page {
         @inject(WebDriver) private readonly webDriver: WebDriver,
     ) {}
 
-    public async create(): Promise<void> {
+    public async create(chromePath?: string): Promise<void> {
         if (isEmpty(this.browser)) {
-            this.browser = await this.webDriver.launch();
+            this.browser = await this.webDriver.launch(chromePath);
```
Comment (haonliu) [inline packages/cli/src/scanner/page.ts]: I feel like chrome path should be a property of the web driver. Maybe we should pass it into the constructor of WebDriver?

Comment (AhmedAbdoOrtiga) [reply packages/cli/src/scanner/page.ts]: No, we only wanna expose it to the library, and we don't want any breaking changes in the CLI

Comment (devaradhanm) [reply packages/cli/src/scanner/page.ts]: @haonliu, we may need to refactor this to may be look something like this - https://github.com/devaradhanm/hello-world-javascript-action/blob/master/browser/web-driver.ts

Comment (devaradhanm) [reply packages/cli/src/scanner/page.ts]: but we can do that later

File: packages/cli/src/web-driver/web-driver.ts [hunk 2]
```diff
@@ -9,12 +9,19 @@ export class WebDriver {
...
+        let launchOption: Puppeteer.LaunchOptions;
+        launchOption = {
             headless: true,
             timeout: 30000,
             args: ['--disable-dev-shm-usage'],
-        });
+        };
+
+        if (chromePath !== undefined) {
+            launchOption.executablePath = chromePath;
```
Comment (devaradhanm) [inline packages/cli/src/web-driver/web-driver.ts]: codecov says you don't have test for this

**Pull request merged**